test/smoke/package.json

```diff
@@ -22,7 +22,7 @@
     "gulp-untar": "^0.0.7",
     "gulp-vinyl-zip": "^2.1.2",
     "mkdirp": "^0.5.1",
-    "mocha": "^3.2.0",
+    "mocha": "^6.0.2",
```

Comment:
Please make sure that mocha will not hang during the tests due to this change
https://boneskull.com/mocha-v4-nears-release/#mochawontforceexit

Reply:
@SounD120 done 😃